src/toxcore/state_format/old.rs

```diff
@@ -151,6 +160,37 @@ pub struct NospamKeys {
 /// Number of bytes of serialized [`NospamKeys`](./struct.NospamKeys.html).
 pub const NOSPAMKEYSBYTES: usize = NOSPAMBYTES + PUBLICKEYBYTES + SECRETKEYBYTES;
 
+/** E.g.
```

Comment:
The problem is if you just throw source code at me in the documentation, I might as well read the actual code. To make the documentation a documentation it should contain some words about how this works and why it works the way it works.

If my Rust isn't too rusted (*badum tss*) the documentation for this should be something like "The default Nospamkeys is never all '0' and is different for each invocation".